android/measure/src/main/java/sh/measure/android/config/MeasureConfig.kt

```diff
@@ -188,6 +189,16 @@ class MeasureConfig(
      * through the app.
      */
     override val trackFragmentLoadTime: Boolean = DefaultConfig.TRACK_FRAGMENT_LOAD_TIME,
+
+    /**
+     *  Map of HTTP headers to be included in http request.
+     *  Defaults to an empty map. The following headers are always excluded:
+     *  * "Content-Type"
+     *  * "msr-req-id"
+     *  * "Authorization"
+     *  * "Content-Length"
+     */
+    override val requestHeadersProvider: MsrRequestHeadersProvider = DefaultConfig.MSR_REQUEST_HEADER_PROVIDER,
```

Comment:
I'd like this to by a `nullable` field, to avoid an unnecessary object from being created. The current default implementation would also create a new empty map each time a network request is made.